rasa/core/agent.py

```diff
@@ -663,51 +527,7 @@ async def handle_text(
 
         msg = UserMessage(text_message.get("text"), output_channel, sender_id)
 
-        return await self.handle_message(msg, message_preprocessor)
-
-    def load_data(
```

Comment:
Wohoo! 💯 I wanna remove this from `Agent` since 2 years and now we're doing it! 💪🏻 🥇